app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -41,17 +33,21 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
+
             RentMinderTheme {
                 Surface(
-                    modifier = Modifier.fillMaxSize(),
-                    color = MaterialTheme.colors.background
+                    modifier = Modifier.size(2800.dp).padding(end = 5.dp),
+                    color = MaterialTheme.colors.background,
```

Comment:
fillMaxSize() is more flexible than hardcoded layout.  Hardcoded dimensions won't make sense on all layouts.  Leave this as it were.

Reply:
ok, I will change that, thank you